docs/how-to/default-profile.md

```diff
@@ -74,7 +74,7 @@ Not all profiles are compatible with each other simply because they use very dif
 
 ### MacOS
 
-We will download the tarball for MacOS, extract the contents, and move the binary file to our path
+You download the tarball for MacOS, extract the contents, and move the binary file to your path:
```

Comment:
```suggestion
Download the tarball for MacOS, extract the contents, and move the binary file to your path:
```

Let's turn this into an instruction rather than a cometary.